dir.props

```diff
@@ -189,7 +189,7 @@
   <PropertyGroup>
     <!-- Possible Values: win7 / ubuntu14.04 / osx.10.10 -->
     <RuntimeSystem Condition="'$(TargetPlatformIdentifier)' == 'Windows_NT'">win7</RuntimeSystem>
-    <RuntimeSystem Condition="'$(TargetPlatformIdentifier)' == 'Unix'">ubuntu.14.04</RuntimeSystem>
+    <RuntimeSystem Condition="'$(RuntimeSystem)' == '' and '$(TargetPlatformIdentifier)' == 'Unix'">ubuntu.14.04</RuntimeSystem>
```

Comment:
Yeah I originally was scared of removing it because I didn't want to break anything else but after I searched more I can see it's not used anywhere else.  I'll remove it.